tests/test_preload_script.py

```diff
@@ -95,3 +95,146 @@ async def test_addPreloadScript_logging(websocket, context_id):
     # Assert navigation is finished.
     result = await read_JSON_message(websocket)
     assert result == {"id": command_id, "result": ANY_DICT}
+
+
+@pytest.mark.asyncio
+async def test_addPreloadScript_loadedInMultipleContexts(
```

Comment:
I am not sure if there's much value in this test. Let me know if I should remove it.